csuntechlab/iSTART
# STR-147 -- FE: Implement Information Slides #26, #28, #29, #30, #33

# Description
Implemented informational slides.

# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. Visit alcohol module and ensure slides ***26, 28, 29, 30, 30*** are implemented.
3. Check for visual bugs and proper placement.
